libs/events/package.json

```diff
@@ -1,12 +1,23 @@
 {
   "name": "@cowprotocol/events",
-  "version": "0.0.1",
-  "main": "./index.js",
-  "types": "./index.d.ts",
+  "version": "0.1.0-RC.1",
```

Comment:
Once I get approvals, I will close versions for the 4 libs published here: types, events, and the 2 widget libs